portal/src/graphql/adminapi/UsersList.tsx

```diff
@@ -271,53 +326,78 @@ const UsersList: React.VFC<UsersListProps> = function UsersList(props) {
   );
 
   const onRenderUserItemColumn = useCallback(
+    // eslint-disable-next-line complexity
     (item: UserListItem, _index?: number, column?: IColumn) => {
       switch (column?.key) {
         case "info": {
           return <UserInfo item={item} />;
         }
         case "action": {
-          const theme =
+          const variant =
             item.deleteAt != null
-              ? themes.actionButton
+              ? "actionButton"
               : item.isDisabled
-              ? themes.actionButton
-              : themes.destructive;
+              ? "actionButton"
+              : "destructive";
 
-          const children =
-            item.deleteAt != null ? (
-              <FormattedMessage id="UsersList.cancel-removal" />
-            ) : item.isAnonymized ? null : item.anonymizeAt != null ? (
-              <FormattedMessage id="UsersList.cancel-anonymization" />
-            ) : item.isDisabled ? (
-              <FormattedMessage id="UsersList.reenable-user" />
-            ) : (
-              <FormattedMessage id="UsersList.disable-user" />
-            );
+          const text =
+            item.deleteAt != null
+              ? renderToString("UsersList.cancel-removal")
+              : item.isAnonymized
+              ? ""
+              : item.anonymizeAt != null
+              ? renderToString("UsersList.cancel-anonymization")
+              : item.isDisabled
+              ? renderToString("UsersList.reenable-user")
+              : renderToString("UsersList.disable-user");
```

Comment:
Actually `variant` and `text` are coupled to each other, they should be generated in the same logic.

```typescript
let variant
let text

if  (item.deleteAt != null) {
  variant = "default"
  text = renderToString("UsersList.cancel-removal")
} else if (item.anonymizeAt != null) {
  variant = "default"
  text = renderToString("UsersList.cancel-anonymization")
} else if () { ... }
```